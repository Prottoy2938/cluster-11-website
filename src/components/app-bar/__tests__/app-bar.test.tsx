import React from "react";
import { render, cleanup } from "@testing-library/react";
import AppBar from "../app-bar";
afterEach(cleanup);

test("should take and match snapshot", () => {
  const { asFragment } = render(<AppBar />);
  expect(asFragment(<AppBar />)).toMatchSnapshot();
});
