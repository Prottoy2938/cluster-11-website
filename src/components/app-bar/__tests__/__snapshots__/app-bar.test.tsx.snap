// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take and match snapshot 1`] = `
<DocumentFragment>
  <div
    class="root"
  >
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary container MuiPaper-elevation4"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <h6
          class="MuiTypography-root title MuiTypography-h6 MuiTypography-noWrap"
        >
          Cluster 11
        </h6>
        <div
          class="searchContainer"
        >
          <div
            class="search"
          >
            <div
              class="searchIcon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                />
              </svg>
            </div>
            <div
              class="MuiInputBase-root inputRoot"
              spellcheck="false"
            >
              <input
                aria-label="search"
                autocomplete="off"
                class="MuiInputBase-input inputInput"
                placeholder="Searchâ€¦"
                type="text"
                value=""
              />
            </div>
          </div>
          <ul
            class="suggestionContainerClosed"
          />
        </div>
      </div>
    </header>
  </div>
</DocumentFragment>
`;
